### PR TITLE
Update Osnabrück locality and county

### PR DESCRIPTION
**Pull request by pathmapper**
Fixes whosonfirst-data/whosonfirst-data#1697, replaces https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/13

### CODE REVIEW

File: data/102/063/901/102063901.geojson [hunk 1]
```diff
@@ -153,59 +147,58 @@
...
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "hasc:id":"DE.NI.OL",
-        "wd:id":"Q5940"
+        "hasc:id":"DE.NI.OB",
+        "wd:id":"Q2916"
     },
-    "wof:coterminous":[
-        101748795
```
Comment (stepps00) [inline data/102/063/901/102063901.geojson]: the `101748795 ` record will also have a `wof:coterminous` property with `102063901` listed. Since we're removing the property in this record, `101748795`will also need to updated to remove the property.

**Comment by stepps00** [comment]
Thank you for the PR, @pathmapper!

One nit about the `wof:coterminous` property, but overall, this looks great! Once we untangle that coterminous issue, this looks good to merge.

File: data/102/063/901/102063901.geojson (hunk 1 above)
Comment (pathmapper) [reply data/102/063/901/102063901.geojson]: Good point!

Just noticed that `101748795` and `102063901` (from this PR) are both records for the locality Osnabrück (= the district free city), identical geometries that serve the **same** functions. I think there is only one needed, or do you want to keep both?

How to proceed? Remove the `wof:coterminous` also from `101748795` to merge this PR and then work on the duplicated record issue?

Comment (stepps00) [reply data/102/063/901/102063901.geojson]: Sorry, this got lost in the shuffle... a few things to address here:

First, WOF maintains continuous coverage of locality, localadmin, and county placetypes in Germany, all by way of the [Bundesamt für Kartographie und Geodäsie](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/de-bkg.json).

In the case of the "city" of Osnabrück, WOF maintains:

- county: https://spelunker.whosonfirst.org/id/102063901/
- localadmin: https://spelunker.whosonfirst.org/id/1377688121/
- locality: https://spelunker.whosonfirst.org/id/101748795/

For the "rural district" of Osnabrück, WOF maintains this county:

- https://spelunker.whosonfirst.org/id/102063947/

So, we should not be adjusting any placetypes here, as we have continuous coverage at each placetype. Also note that in WOF, we don't change the placetype of a record - if the placetype is incorrect, then we deprecate the record and supersede it into a _new_ record. But, in this case, we already have records at each placetype. We should also continue to maintain the coterminous property, as these records represent the same place.

I agree that there are some wonky statoids properties that need to be adjusted on these records, but the names, labels, and hierarchies all look correct for these records.

So in summary, I think it's best to only adjust the statoids properties in these records. What are your thoughts?

Comment (pathmapper) [reply data/102/063/901/102063901.geojson]: Thanks for listing the different records and the explanation @stepps00.

Some of my confusion was related to the fact that in WOF there is a county Osnabrück `102063901` with the same geometry as the locality `101748795`. The city of Osnabrück is in German: _Kreisfreie Stadt_ which means _City which doesn't belong to a rural district/county_. But if this is how WOF is organized, thats fine and the coterminous property for both makes sense.

However, besides the statoids properties there are also some other properties which I think need to be adjusted, e.g. for https://spelunker.whosonfirst.org/id/102063947/:

This is the Landkreis Osnabrück, the real existing rural district which is different from the city.
So these properties are not correct:
**label**
deu_x_preferred_longname | Kreisfreie Stadt Osnabrück
deu_x_preferred_placetype | kreisfreie stadt
eng_x_preferred_placetype | district-free city

**name**
eng_x_preferred | Oldenburg (totally different name of another County and City)

**wof**
concordances | hasc:id | DE.NI.OS (according to statoids it's DE.NI.ON)
placetype_local | district-free city (this is the rural district, not the the city)

For https://spelunker.whosonfirst.org/id/102063901/ 
all **label** properties are not correct
a lot of the **name** properties are not correct (it's not a rural disctrict)
**wof**: It should DE.NI.OB instead of DE.NI.OL
and so on 

Do you agree?

Comment (stepps00) [reply data/102/063/901/102063901.geojson]: Yep - that all sounds correct!

For both of these records, the label properties come from `statoids:` properties.. and since the statoids properties are incorrect, we should correct both the `statoids:` and label properties. I'm not surprised that there are other name properties that need fixing, too.

So, it sounds like adjustments are only needed to the `name:*`, `statoids:*`, and `label:*` properties. Is this something you're able to tackle in this PR or would you like to create a new PR with changes? Do you need any assistance in making these changes?

Comment (pathmapper) [reply data/102/063/901/102063901.geojson]: > Is this something you're able to tackle in this PR

Yep

---

For me it looks like there is a need to adjust also some `wof:` properties, e.g. for https://spelunker.whosonfirst.org/id/102063947/:

If I do the needed adjustments only for `name:*`, `statoids:*`, and `label:*` properties and run the exportify tool in the GeoJSON afterwards, the following `wof:` properties are not looking correct:

- `wof:concordances`:`hasc:id`: "DE.NI.OS" (should be DE.NI.ON according to statoids, or is there another source for this)
- `wof:placetype_local`: "district-free city" (should be "rural district")
- `wof:population`: 157267 (should be 350147 according to statoids)
- `wof:population_rank`: 9 (should be 10 if the population would be 350147)
- `wof:shortcode`: "OS" (should be ON according to statoids, or is there another source for this)

In https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/17#discussion_r341267640 you mentioned:

> Also note that in WOF, we don't change the placetype of a record - if the placetype is incorrect, then we deprecate the record and supersede it into a new record. 

Does this account also for `wof:placetype_local`?

Comment (stepps00) [reply data/102/063/901/102063901.geojson]: Yep - that all looks correct.. as far as `wof:placetype_local`, we can adjust that, since it is sourced from the Statoids property.

**Comment by pathmapper** [comment]
Reverted the unwanted changes in c28c993, should be good to merge now.

@stepps00 please let me know if this is not the case:-)

**Comment by pathmapper** [comment]
@stepps00 anything left to do here?

**Comment by stepps00** [comment]
@pathmapper - apologies, this was lost in the shuffle. Edits looks good, so I'll merge. Thank you!

**Comment by pathmapper** [comment]
Thanks @stepps00, no worries :)
